test/NNPDE_tests.jl

```diff
@@ -553,3 +553,35 @@ p_ = res.minimizer[end-2:end]
 # u_predict  = [[discretization.phi[i]([t],minimizers[i])[1] for t in ts] for i in 1:3]
 # plot(sol)
 # plot!(ts, u_predict, label = ["x(t)" "y(t)" "z(t)"])
+
+@parameters t
+@variables i(..)
+Di = Differential(t)
+Ii = Integral(t, DomainSets.ClosedInterval(0, t))
+eq = Di(i(t)) + 2*i(t) + 5*Ii(i(t)) ~ 1
+bcs = [i(0.) ~ 0.0]
+domains = [t ∈ Interval(0.0,2.0)]
+chain = FastChain(FastDense(1,15,Flux.σ),FastDense(15,1))
```

Comment:
it's some trouble with `FastChain` after the new release of `Flux`, just use `Chain` for a test while `FastChain` won't have been fixed.

Reply:
what's up?

Reply:
`initial_params` change behavior after the update in Flux and now it works incorrectly.
```julia
chain = FastChain(FastDense(1,12,Flux.σ),FastDense(12,1))
julia> initθ = DiffEqFlux.initial_params(chain)
26-element Vector{Union{Nothing, Float32}}:
  0.44163412f0
  0.3313627f0
 -0.24543865f0
  0.16519924f0
  0.18937507f0
 -0.047894925f0
  ⋮
 -0.50703824f0
 -0.13543157f0
  0.036089323f0
  0.0010503983f0
   nothing
```

Reply:
Can you try and narrow that down?

Reply:
ok, I will open the issue in DiffEqFlux as soon as I figure out what exactly is the problem with `DiffEqFlux.initial_params`.